packages/nimble-components/src/rich-text-mention/users/tests/rich-text-mention-users.stories.ts

```diff
@@ -25,6 +25,15 @@ The object's type is \`RichTextMentionValidity\`, and it contains the following
 -   \`missingDisplayNameValue\`: \`true\` when a mapping has no \`display-name\` value
 `;
 
+const mentionUpdateEventDescription = `This event will be fired on following action:
+
+- Whenever the \`@\` character is entered into the editor
+- When the user types any character after \`@\` into the editor
+
+This fires with the \`eventData\` containing the current text that is added after the \`@\` character and before the current position of the
+    text cursor.
+`;
+
```

Comment:
This description appears in the rich text viewer documentation too, but the actions described don't apply to the viewer. Could you update it to distinguish how the event behaves in both components? I'm not certain how that is but it could be something like:
```suggestion
const mentionUpdateEventDescription = `For the editor, this event will be fired on following action:

- Whenever the \`@\` character is entered into the editor
- When the user types any character after \`@\` into the editor

This fires with the \`eventData\` containing the current text that is added after the \`@\` character and before the current position of the
    text cursor.

For the viewer, this event will never be fired.
`;

```

Reply:
Updated